.github/workflows/release.yml

```diff
@@ -85,19 +85,19 @@ jobs:
           key: cargo-${{ matrix.target }}-${{ hashFiles('**/Cargo.lock') }}
       - uses: dtolnay/rust-toolchain@1.67
       - name: Build the release binaries
-        run: |
-          rustup target add ${{ matrix.target }}
-          cargo build --target ${{ matrix.target }} --package=radicle-httpd --release
-          cargo build --target ${{ matrix.target }} --package=radicle-node --release
-          cargo build --target ${{ matrix.target }} --bin rad --release
-          cargo build --target ${{ matrix.target }} --bin git-remote-rad --release
-      - shell: bash
         run: |
           staging="radicle-${{ matrix.target }}"
           mkdir -p "$staging"
+          rustup target add ${{ matrix.target }}
+          cargo build --target ${{ matrix.target }} --package=radicle-httpd --release
           cp target/${{ matrix.target }}/release/radicle-httpd "$staging"/
+          cargo build --target ${{ matrix.target }} --package=radicle-node --release
           cp target/${{ matrix.target }}/release/radicle-node "$staging"/
+          cargo build --target ${{ matrix.target }} --bin rad --release
           cp target/${{ matrix.target }}/release/rad "$staging"/
+          cargo build --target=${{ matrix.target }} --bin rad
+          cp target/${{ matrix.target }}/debug/rad "$staging"/rad-debug
+          cargo build --target ${{ matrix.target }} --bin git-remote-rad --release
```

Comment:
Could we perhaps split these into two build steps? Relase and Debug?

Reply:
Really? Can't we just have it as a variable in the build matrix?

Reply:
We’d need a bit more than a variable: a conditional, and those are messy with the build matrix